lib/matplotlib/testing/decorators.py

```diff
@@ -518,3 +519,28 @@ def _image_directories(func):
     result_dir = Path().resolve() / "result_images" / module_path.stem
     result_dir.mkdir(parents=True, exist_ok=True)
     return baseline_dir, result_dir
+
+
+needs_ghostscript = pytest.mark.skipif(
+    "eps" not in matplotlib.testing.compare.converter,
+    reason="This test needs a ghostscript installation")
```

Comment:
Rather than a decorator, should these be turned into a "marker"? Then you'd decorate the tests with:
`@pytest.mark.needs_lualatex` and avoid the decorator imports.

Reply:
How exactly would that work? You'd evaluate whether gostscript is present at test startup and then modify the marker selection? I don't know how to do that but I guess that's possible. However, importing the decorator seems much simpler.

Reply:
I wasn't sure how easy it would be myself, just figured it would be worth the thought :) I think we could intercept the setup of the test and say something like:
```python
if needs_ghostscript in item.markers and "eps" not in matplotlib.testing.compare.converter:
    pytest.skip(reason=...)
```
https://docs.pytest.org/en/6.2.x/example/markers.html#marking-platform-specific-tests-with-pytest

I agree though, that might be more than necessary here.